pkg/api/v1beta3/types.go

```diff
@@ -885,6 +894,9 @@ type Node struct {
 
 	// Status describes the current status of a Node
 	Status NodeStatus `json:"status,omitempty"`
+
+	// Info are information collected on the Node
+	Info NodeInfo `json:"info,omitempty"`
```

Comment:
This violates the v1beta3 convention. NodeInfo cannot go here. It must go into NodeStatus.